aggregator/src/aggregator/http_handlers/tests/aggregation_job_init.rs

```diff
@@ -734,28 +735,23 @@ async fn aggregate_init_with_reports_encrypted_by_task_specific_key() {
         aggregation_param,
     );
 
-    // Same ID as the task to test having both keys to choose from.
-    let global_hpke_keypair_same_id =
-        HpkeKeypair::test_with_id((*helper_task.current_hpke_key().config().id()).into());
-    datastore
-        .run_unnamed_tx(|tx| {
-            let global_hpke_keypair_same_id = global_hpke_keypair_same_id.clone();
-            Box::pin(async move {
-                // Leave these in the PENDING state--they should still be decryptable.
-                match tx
-                    .put_global_hpke_keypair(&global_hpke_keypair_same_id)
-                    .await
-                {
-                    // Prevent test flakes in case a colliding key is already in the datastore.
-                    // The test context code randomly generates an ID so there's a chance for
-                    // collision.
-                    Ok(_) | Err(datastore::Error::MutationTargetAlreadyExists) => Ok(()),
-                    Err(err) => Err(err),
```

Comment:
:facepalm: ah I see, there is no `ON CONFLICT DO NOTHING` in `put_global_hpke_keypair()` so if there's a conflict it'll fail with a harder-to-match postgres uniqueness constraint violation error.